build/asset-manifest.json

```diff
@@ -1,13 +1,14 @@
 {
```

Comment:
### Optional
- [ ] It would be better if we capitalize our commit messages, also it would be better if we have commit messages like the one we have in the PR so that when we click on the commit we can see exactly what is done :champagne: